app/org/sagebionetworks/bridge/play/controllers/BaseController.java

```diff
@@ -258,13 +260,25 @@ void verifySupportedVersionOrThrowException(Study study) throws UnsupportedVersi
                 LOG.debug("Malformed Accept-Language header sent: " + acceptLanguageHeader);
             }
         }
+
+        // if no Accept-Language header detected, we shall add an extra warning header
+        Http.Response response = Http.Context.current().response();
+        response.setHeader(BridgeConstants.BRIDGE_API_STATUS_HEADER, BridgeConstants.BRIDGE_WARNING_STATUS);
```

Comment:
What happens if they send a junk accept language header, we should also include a warning;
I think the header needs to be a message: "Warning: Accept-Language header is missing or invalid; filtering based on language not available" and "Warning: User-Agent header is missing or invalid; filtering based on application version not available"